pkg/agent/openflow/client.go

```diff
@@ -279,6 +280,18 @@ type Client interface {
 		dstIP net.IP,
 		outPort uint32,
 		igmp ofutil.Message) error
+
+	// InstallTrafficControlMarkFlows installs the flows to mark the packets for the traffic control rule.
+	InstallTrafficControlMarkFlows(name string, srcOfPorts []uint32, dstOfPort uint32, direction v1alpha2.Direction) error
```

Comment:
I haven't taken a look at the implementation of the 4 methods, but from the perspective of the interface's consumer, it's unclear what the methods do from the function names and the comments and I'm not sure what interfaces I should call to redirect or mirror specific traffic. Could you add more details?